script/install_chromium.sh

```diff
@@ -2,41 +2,51 @@
 # Copyright 2013 The Flutter Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+
+# This script may be run as:
+# $ CHROME_DOWNLOAD_DIR=./whatever script/install_chromium.sh
```

Comment:
At some point in 2023, I'd like to migrate this to a `.dart` script rather than bash :)

Reply:
🎉🙂